easybuild/easyconfigs/g/geopsy/geopsy-3.4.2-foss-2022a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'geopsy'
+version = '3.4.2'
+
+homepage = 'https://www.geopsy.org/'
+description = """Graphical environment to view, organize and process signals. Most of the geophysical file formats are 
+automatically recognized. Geopsy features can be extended through plugin tools. A selection is available here below."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://www.geopsy.org/download/archives/']
+sources = ['geopsypack-src-%(version)s.tar.gz']
+patches = ['qgpcoremath-configure.patch', 'arraycore-configure.patch', 'qgpcoretools-configure.patch']
+checksums = [
+    {'geopsypack-src-3.4.2.tar.gz': '5e0114fe32fe01cb9d46713a201462a8c284c43d9dd6c60afaca23f6d76b621f'},
+    {'qgpcoremath-configure.patch': '530b9cb72424a678c86723f65f0ee28011dfbd5850993f8ff397c8785b37c314'},
+    {'arraycore-configure.patch': '95bcb6500410c6f150865d62542abeaf133b61ae1994d6c39162b16a3981e22c'},
+    {'qgpcoretools-configure.patch': 'fc8a20f55fc9571c1cd2ba24122b5a380095653c37e3f79933f6f0ebdbab97ec'},
+]
+
+dependencies = [
+    ('Qt5', '5.15.5'),
+]
+
+configopts = '-accept-license'
+prefix_opt = '-prefix '
+
+skipsteps = ['sanity_check']
```

Comment:
```suggestion
```
Not sure why there is a sanity check command provided but then skipped as well. Anyway this would need to go fomr `sanity_check` to `sanitycheck` in order to work with the latest version of EB